src/pages/RulePage/RulePage.jsx

```diff
@@ -0,0 +1,23 @@
+import './RulePage.css'
+import React from 'react'
+import { useNavigate } from "react-router-dom";
+
+
+const RulePage = ({closeRuleModal}) => {
+
+  const handlePlay = (id) => {
+    navigate(`/category/${id}`);
+  }
+
+  
+  return (
+
+    <div>
```

Comment:
instead of div you can use react fragments beacuse here div creating extra node in dom which you are not using